tyk-headless/tests/deployment-gw-repset_test.yaml

```diff
@@ -0,0 +1,294 @@
+suite: test headless gateway deployment
+templates:
+  - templates/configmap-gateway.yaml
+  - templates/deployment-gw-repset.yaml
+  - templates/secrets.yaml
+  - templates/secret-certs.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+chart:
+  version: 0.12.0
+tests:
+  - it: should match metadata name with default values
+    template: templates/deployment-gw-repset.yaml
+    asserts:
+      - equal:
+          path: apiVersion
+          value:  apps/v1
+      - equal:
```

Comment:
Instead of `equal` should we use `IsKind` assertion like you did below?

Reply:
updated in [2f2bb5b](https://github.com/TykTechnologies/tyk-helm-chart/pull/245/commits/2f2bb5b89ce19ef9f7b73719dacf47f35c5824f1)